### PR TITLE
docs: updating vtt documentation for trestle author docs

### PR DESCRIPTION
**Pull request by AleJo2995**
## Types of changes

- [ ] Hot fix (emergency fix and release)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation (change which affects the documentation site)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Release (`develop` -> `main`)

## Quality assurance (all should be covered).

- [x] My code follows the code style of this project.
- [x] Documentation for my change is up to date?
- [x] My PR meets testing requirements.
- [x] All new and existing tests passed.
- [x] All commits are signed-off.

## Summary

This documentation is related to issue #1464 about _**enabling validate template type parameter for trestle author docs command**_

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

# Before you merge

- Ensure it is a 'squash commit' if not a release.
- Ensure CI is currently passing
- Check sonar. If you are working for a fork a maintainer will reach out, if required.

### CODE REVIEW

File: docs/trestle_author.md [hunk 1]
```diff
@@ -296,6 +296,61 @@ Running `trestle author docs validate -tn docs_task -gh="Governed section"` will
 - If `--template-version 1.0.0` (`-tv`) is passed the header field `x-trestle-template-version` will be ignored and document will be forcefully validated against template of version `1.0.0`.
   Use this for testing purposes _only_ when you need to validate the document against a specific template. By default the template version will be determined based on `x-trestle-template-version` in the document.
 
+### Validating the documents against different templates
+
+Validation against multiple templates can be done when there is an scenario where you have multiple templates that will have multiple instances. In this particular case you can have a 1:1 relationship between the template and the instance document you are creating out of it, so validation can be performed based on template type and version of that particular template defined in headers.
```
Comment (vikas-agarwal76) [inline docs/trestle_author.md:301]: "when there is an .." "when there is a..."

File: docs/trestle_author.md [hunk 2]
```diff
@@ -296,6 +296,61 @@ Running `trestle author docs validate -tn docs_task -gh="Governed section"` will
...
+
+For that to happen you will need to provide your template with the following parameter at the yaml header level, matching the type of template to be implemented so the validation can occur:
+
```
Comment (vikas-agarwal76) [inline docs/trestle_author.md:304]: "implemented so the ..." "implemented so that the ...."

**Review by vikas-agarwal76 — APPROVED** [review]
LGTM. Just edits.

File: docs/trestle_author.md (hunk 1 above)
Comment (AleJo2995) [reply docs/trestle_author.md]: Done

File: docs/trestle_author.md (hunk 2 above)
Comment (AleJo2995) [reply docs/trestle_author.md:304]: Done

**Pull request merged**